### PR TITLE
Regenerate for Kubernetes 1.31

### PR DESCRIPTION
**Pull request by brendandburns**
@yue9944882 I had to generate this in my own branch, and then it turned out the patches were a little broken, so fixed up the JSON.java patch also. Let me know if you want me to split out that fix into a different branch.

### CODE REVIEW

File: scripts/patches/json.diff [hunk 1]
```diff
@@ -1,62 +1,64 @@
...
-@@ -58,7 +73,10 @@ public class JSON {
+@@ -65,8 +82,11 @@ public class JSON {
      public static GsonBuilder createGson() {
          GsonFireBuilder fireBuilder = new GsonFireBuilder()
          ;
 -        GsonBuilder builder = fireBuilder.createGsonBuilder();
+-        return builder;
 +        GsonBuilder builder =
 +                fireBuilder
 +                        .registerPreProcessor(V1Status.class, new V1StatusPreProcessor())
```
Comment (yue9944882) [inline scripts/patches/json.diff:55]: @brendandburns looks like the patch has been broken for a while, curiously did you change JSON.java manually when generating 1.30?

**Comment by yue9944882** [comment]
/lgtm

@brendandburns the change LGTM, could you squash the change into single commit to workaround the CLA check? it's a bit tricky that we can't do a normal squash, instead we need to do a force reset and re-commit to purge the bot author information from the commit.

**Comment by cache-sk** [comment]
Can I humbly ask for the models to be regenerated again so that there are swagger annotations as well?

Alternatively, regenerate them again after this merge, but before the release of 22.x.y?

Thank you very much in advance.

File: scripts/patches/json.diff (hunk 1 above)
Comment (brendandburns) [reply scripts/patches/json.diff:55]: Yeah, I think I broke this when I regenerated 1.30

**Comment by brendandburns** [comment]
@cache-sk its going to take some work to regenerate with the annotations, I'm going to merge this PR. Then we'll see if I have time to run the regeneration before we cut the release. We will definitely get it into a patch release for 22.0.x

**Comment by brendandburns** [comment]
@yue9944882 squashed, it needs a new lgtm since the commits changed.

**Comment by cache-sk** [comment]
> @cache-sk its going to take some work to regenerate with the annotations, I'm going to merge this PR. Then we'll see if I have time to run the regeneration before we cut the release. We will definitely get it into a patch release for 22.0.x

@brendandburns Can i humbly ask to regenerate model for 23.0.1?

**Pull request merged**